app/cdap/components/shared/ConfirmDialog/index.tsx

```diff
@@ -68,12 +86,25 @@ export const ConfirmDialog = ({
     }
   };
 
+  const showCopyOption = () => {
```

Comment:
why is it named `showCopyOption`? Isn't is doing copy to clipboard?

Reply:
Changed the handling way now.